src/auth-service/utils/user.util.js

```
@@ -4314,6 +4318,147 @@ const createUserModule = {
+              requestType: "group",
+            })
+            .lean();
+
+          const summary = {
+            totalRequestsChecked: approvedRequests.length,
+            usersFixed: 0,
+            usersAlreadyMember: 0,
+            usersNotFound: 0,
+            groupsNotFound: 0,
+            rolesNotFound: 0,
+            errors: [],
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)